src/CommunityToolkit/Xamarin.CommunityToolkit/Views/ViewToRendererConverter.uwp.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Xamarin.CommunityToolkit.Extensions;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.UWP;
+using WRect = Windows.Foundation.Rect;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public class ViewToRendererConverter
+	{
+		// This class is ported from Xamarin.Forms and should remain in sync
+		internal class WrapperControl : Panel
+		{
+			readonly View _view;
```

Comment:
Note for reviewers, since this a ported code (basically the maintenance will be copy and paste), I think that is ok to keep the same code style as the original source. If you agree with me, we can suppress the warning in this file.